### ISSUE TITLE
Support for Multiple Commodity conversion prices

### ISSUE THREAD

**Issue by hawkpro-1** [marek-g/rust-ledger-parser#11]
First of all thank you for the parser, its amazing to find a parser written in rust :) 
It would be great to see support for multiple commodities which is usually the case when dealing with stocks and other assets 
eg:

```
2020-02-01
    assets:cc:ada          2000 ADA @ $0.02
    assets:bank:checking                   $-40
```
from https://hledger.org/investments.html?highlight=stock#a-stock-purchase

thanks :D

**Comment by marek-g** [comment]
Thank you hwkpro-1! I agree, it's a very nice feature to have, so yes, it's on my track.

**Comment by marek-g** [comment]
Implemented in version 5.0.0:

https://github.com/marek-g/rust-ledger-parser/commit/8c063adf20d40d74a3f0586c16de524e7b3c73dc
